crates/ef-testing/src/models/result.rs

```diff
@@ -11,6 +12,17 @@ pub(crate) fn log_execution_result(
                 warn!("{} reverted:\n{}", case_name, err.replace("\\n", "\n"));
             } else {
                 info!("{} passed: {:?}", case_name, info.actual_resources);
+                if let Some(call) = info.execute_call_info {
```

Comment:
you need to check the status of the tx, see for example https://github.com/kkrt-labs/kakarot/blob/66acb17d0500d70f4f0ce81acaa5f8bb7b4632ee/scripts/utils/kakarot.py#L332